src/components/forms/Registration.tsx

```diff
@@ -145,22 +145,22 @@ const RegistrationForm = ({
     } = values;
     const params = {
       eventTime,
-      phone,
+      phone: phone || undefined,
       geom,
       assignedTo: assignedTo?.id,
       stockingCustomer: stockingCustomer?.id,
       location: location,
-      fishOrigin,
+      fishOrigin: fishOrigin.toUpperCase(),
       fishOriginCompanyName,
       ...(fishOriginReservoir && {
         fishOriginReservoir,
       }),
       batches: batches.map((batch) => {
         return {
-          amount: batch.amount,
-          weight: batch.weight,
-          fishType: batch?.fishType?.id,
-          fishAge: batch?.fishAge?.id,
+          amount: batch.amount || undefined,
```

Comment:
o nebuna default "undefined" , jei kažkas negrįžta? ar čia dėl to kad null gali būti ar dar kokia kita falsy value ir tada undefined turi buti?  Jei taip tai ok, galim mergint

Reply:
Gali buti tuscias stringas is formos.